packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -46,12 +46,15 @@
       white-space: normal;
       word-wrap: break-word; /* For IE 10 and IE 11 */
 
-      &:hover {
-        background-color: var(--terra-paginator-nav-link-hover-background-color, transparent);
-        border: var(--terra-paginator-nav-link-hover-border, none);
-        box-shadow: var(--terra-paginator-nav-link-hover-box-shadow);
-        color: var(--terra-paginator-nav-link-hover-color, #001f67);
-        text-decoration: var(--terra-paginator-nav-link-hover-text-decoration, underline);
+      /* stylelint-disable max-nesting-depth */
```

Comment:
Can we disable next line or re-enable this rule after this query?

Reply:
Done 9892dee

Reply:
Can we apply this to any place `/* stylelint-disable max-nesting-depth */` is used? I noticed quite a few still need cleaned up